src/main/java/nl/knaw/dans/ingest/core/service/mapper/mapping/SpatialPoint.java

```diff
@@ -36,4 +37,24 @@ public class SpatialPoint extends Spatial {
         builder.addSubfield(SPATIAL_POINT_X, point.getX());
         builder.addSubfield(SPATIAL_POINT_Y, point.getY());
     };
+
+    public static boolean isPoint(Node node) {
+        var hasChild = hasChildNode(node, "gml:Point/gml:pos");
+        var srsName = Base.getAttribute(node, "srsName")
+            .map(Node::getTextContent)
+            .filter(StringUtils::isNotBlank)
+            .orElse(null);
+
+        if (!hasChild) {
+            return false;
+        }
+
+        // TS002: if @srsName=http://www.opengis.net/def/crs/EPSG/0/28992
+        if (RD_SRS_NAME.equals(srsName)) {
+            return true;
+        }
+
+        // TS003: if @srsName not present OR @srsName=http://www.opengis.net/def/crs/EPSG/0/4326
+        return srsName == null || LONLAT_SRS_NAME.equals(srsName);
```

Comment:
When we don't make it mandatory in the validator as [discussed](https://drivenbydata.atlassian.net/browse/DD-1305?focusedCommentId=46542) with @lindareijnhoudt 

at least 

```suggestion
        // TS002: if @srsName=http://www.opengis.net/def/crs/EPSG/0/28992
        // TS003: if @srsName not present OR @srsName=http://www.opengis.net/def/crs/EPSG/0/4326
        return srsName == null || LONLAT_SRS_NAME.equals(srsName) || RD_SRS_NAME.equals(srsName);
```

Reply:
Don't want to be nitpicky but I figured having these as separate statements makes its easier to reason about the rules. But just adding them as comments might also be good